src/app/contexts/RequestContext/isNotUK/index.test.js

```diff
@@ -0,0 +1,34 @@
+import isNotUK from './index';
+
+describe('isNotUK', () => {
+  describe('undefined input', () => {
+    it('should return true', () => {
+      expect(isNotUK()).toEqual(true);
```

Comment:
Are we sure we want this to default to true?

Reply:
Yes, I think it should. 

I understand the reason for changing `isUK` to `isNotUk`, is because we'd rather assume the user is not in the UK, if we are not sure where the user is, than assume the user is in the UK.

Reply:
>if we are not sure where the user is, than assume the user is in the UK.

Unless I am mistaken this test shows the opposite to your statement?

I think this is saying if we do not know where the user is assume they are not in the UK

Reply:
> if we are not sure where the user is, than assume the user is in the UK.

Sorry, this should be "then assume the user is NOT in the UK."

Reply:
That selection is a little confusing when read as a typo, because the preceding part is important.

I'm saying we'd rather assume a user is not in the UK, if it's not clear where the user is.

So, the `isNotUK` function, which accepts a country code, should return `true`, if no country code is provided, or if the supplied country code is `null`, `undefined`, or an empty string.

Reply:
@jimjohnsonrollings @neildoughty Can I just double check the above with you both please.

if we are unable to detect where a user is i.e. we have n o country code or the country code is invalid should we then presume a user is in the uk or outside of the UK?

This could be very important if we start to use this logic for things like cookie banners etc.

Reply:
This was complicated when we tried it before.  Essentially we have to follow the same logic that other people use I think, which is to assume not-UK.

We originally tried to assume people were in the UK, but this conflicted with what upstream systems do already.

Reply:
the last time i checked the uk audience for ws was about 1.5% of the total.

so if we're presuming non-UK, and then updating the information when we know better that seems logical.

it's prob more likely we will not be able to detect users' location on old devices, who are more likely to be out of the UK, right?

Reply:
Agree on both points. Thanks all.